htdocs/public/partnership/new.php

```diff
@@ -211,17 +217,64 @@ function llxFooterVierge()
 			$partnership->fk_soc = 0;
 		}
 
-		$partnership->statut      = -1;
-		$partnership->firstname   = GETPOST('firstname');
-		$partnership->lastname    = GETPOST('lastname');
-		$partnership->address     = GETPOST('address');
-		$partnership->zip         = GETPOST('zipcode');
-		$partnership->town        = GETPOST('town');
-		$partnership->email       = GETPOST('email');
-		$partnership->country_id  = GETPOST('country_id', 'int');
-		$partnership->state_id    = GETPOST('state_id', 'int');
-		//$partnership->typeid      = $conf->global->PARTNERSHIP_NEWFORM_FORCETYPE ? $conf->global->PARTNERSHIP_NEWFORM_FORCETYPE : GETPOST('typeid', 'int');
+		$partnership->status      = 0;
 		$partnership->note_private = GETPOST('note_private');
+		$partnership->date_creation = dol_now();
+		$partnership->date_partnership_start = dol_now();
+		$partnership->fk_type = $partnershipt->fetch(0, 'default');
```

Comment:
The fetch may return a negative value if error or not found, so split the line in 2.
First 
$partneshipt->fetch 
then 
if ($partnershipt->id > 0) {
   $partnership->fk_type = $partnershipt->id
}
}